SeaPublicWebsite.BusinessLogic/SeaPublicWebsite.BusinessLogic.csproj

```
@@ -1,8 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
         <ImplicitUsings>enable</ImplicitUsings>
+        <LangVersion>11</LangVersion>

```

Expecting this to be C# 12
```suggestion
        <LangVersion>12</LangVersion>
```